src/solver/conv_hip_implicit_gemm_bwd_data_xdlops.cpp

```diff
@@ -0,0 +1,392 @@
+
+    float elapsed_time =
+        invoker_ptr->Run(argument_ptr.get(), {handle.GetStream(), enable_profiling});
+    if(enable_profiling)
+    {
+        handle.ResetKernelTime();
+        handle.AccumKernelTime(elapsed_time);
+    }
+}
+#endif
+
+void PerformanceConfigHipImplicitGemmBwdXdlops::HeuristicInit(const ProblemDescription& problem)
+{
+    this->index = 0;
+#if !MIOPEN_BACKEND_HIP || !MIOPEN_USE_COMPOSABLEKERNEL
+    std::ignore = problem;
+#else
+    this->index      = 0;
+    this->total_size = 0;
+    this->kernel_id  = "";
+    switch(problem.conv_problem.GetInDataType())
+    {
+    case miopenHalf: Init<ck::half_t>(problem); break;
+    case miopenFloat: Init<float>(problem); break;
+    case miopenInt8:
+    case miopenInt32:
+    case miopenInt8x4:
+    case miopenBFloat16:
+    case miopenDouble: break;
+    }
+#endif
+}
+
+bool PerformanceConfigHipImplicitGemmBwdXdlops::SetNextValue(const ProblemDescription& problem)
+{
+    if(total_size == -1)
+        this->HeuristicInit(problem);
+    assert(total_size != -1);
+    if((index + 1) < total_size)
+    {
+        ++index;
+        return true;
+    }
+    else
+        return false;
+}
+
+bool PerformanceConfigHipImplicitGemmBwdXdlops::IsValidValue() const { return index < total_size; }
+
+bool PerformanceConfigHipImplicitGemmBwdXdlops::IsValid(const ProblemDescription& problem) const
+{
+#if !MIOPEN_BACKEND_HIP || !MIOPEN_USE_COMPOSABLEKERNEL
+    std::ignore = problem;
+    return false;
+#else
+    switch(problem.conv_problem.GetInDataType())
+    {
+    case miopenHalf: return CheckIsSupportCKArgs<ck::half_t>(problem);
+    case miopenFloat: return CheckIsSupportCKArgs<float>(problem);
+    case miopenInt8:
+    case miopenInt32:
+    case miopenInt8x4:
+    case miopenBFloat16:
+    case miopenDouble: break;
+    }
+    return false;
+#endif
+}
+
+bool PerformanceConfigHipImplicitGemmBwdXdlops::operator==(
+    const PerformanceConfigHipImplicitGemmBwdXdlops& other) const
+{
+    return this->index == other.index;
+}
+
+PerformanceConfigHipImplicitGemmBwdXdlops
+ConvHipImplicitGemmBwdXdlops::GetDefaultPerformanceConfig(const ProblemDescription& problem) const
+{
+    PerformanceConfigHipImplicitGemmBwdXdlops pp;
+    pp.HeuristicInit(problem);
+    return pp;
+}
+
+bool ConvHipImplicitGemmBwdXdlops::IsValidPerformanceConfig(
+    const ProblemDescription& problem,
+    const PerformanceConfigHipImplicitGemmBwdXdlops& config) const
+{
+    return config.IsValid(problem);
+}
+
+PerformanceConfigHipImplicitGemmBwdXdlops
+ConvHipImplicitGemmBwdXdlops::Search(const ConvolutionContext& ctx,
+                                     const ProblemDescription& problem,
+                                     const AnyInvokeParams& invoke_ctx) const
+{
+    return GenericSearch(*this, ctx, problem, invoke_ctx);
+}
+
+size_t ConvHipImplicitGemmBwdXdlops::GetWorkspaceSize(const ConvolutionContext& ctx) const
+{
+    std::ignore = ctx;
+    return 0;
+}
```

Comment:
@iq136boy Declaration and definition of `ConvHipImplicitGemmBwdXdlops::GetWorkspaceSize()` can be omitted because default `GetWorkspaceSize()` returns 0